LLama/Native/NativeApi.Load.cs

```diff
@@ -36,6 +37,15 @@ static NativeApi()
 
             // Init llama.cpp backend
             llama_backend_init();
+
+            // Set flag to indicate that this has been done. No native library config can be done after this point.
+            NativeLibraryConfig.LibraryHasLoaded = true;
+
+            // Now that the "loaded" flag is set, configure logging in llama.cpp
+            if (NativeLibraryConfig.Instance.LogCallback != null)
+                NativeLogConfig.llama_log_set(NativeLibraryConfig.Instance.LogCallback);
+            if (NativeLibraryConfig.Instance.LoggerCallback != null)
+                NativeLogConfig.llama_log_set(NativeLibraryConfig.Instance.LoggerCallback);
         }
```

Comment:
There is still a problem here with logs from `llama_backend_init` call: they won't be intercepted. So i believe `llama_backend_init` call should appear after logging initialization. Also hope that `llama_empty_call` won't log anything.

Reply:
That's a good point. I think moving `llama_backend_init();` below the rest of the code here should catch that.

Reply:
The only call above logging is now `llama_empty_call`. That shouldn't ever log anything (it's specifically selected as a method that does nothing, just to trigger DLL loading).